app/geolocation/geolocation.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var geocoder = require('geocoder')
+
+module.exports.submitLoc = function submitLoc (req, res, next) {
+  var newlat = parseFloat(req.body.lat);
+  var newlong = parseFloat(req.body.long);
+
+  geocoder.reverseGeocode(newlat, newlong, function(err, data) {
```

Comment:
The goal of geolocation is to get (approximate) coordinates, why are you using geocoder when you already have the coords?